push.go

```diff
@@ -54,6 +55,11 @@ func (push *pushSocket) Type() SocketType {
 	return push.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.